experimental/requirements.txt

```diff
@@ -1,2 +1,2 @@
 torch>=1.10
-triton==2.0.0.dev20220206
+triton==2.0.0
```

Comment:
hmm this does not actually work, as of now at least (this pip package does not exist yet). Do you have a plan here ? Sync with @ptillet for the release ?